packages/@aws-cdk/aws-servicecatalog/lib/portfolio.ts

```diff
@@ -0,0 +1,88 @@
+import * as cdk from '@aws-cdk/core';
+import { CfnPortfolio } from './servicecatalog.generated';
+
+// keep this import separate from other imports to reduce chance for merge conflicts with v2-main
+// eslint-disable-next-line no-duplicate-imports, import/order
+import { Construct } from 'constructs';
+
+/**
+ * A Service Catalog portfolio.
+ */
+export interface IPortfolio extends cdk.IResource {
+  /**
+   * The ARN of the portfolio.
+   *
+   * @attribute
+   */
+  readonly portfolioArn: string;
+
+  /**
+   * The ID of the portfolio.
+   *
+   * @attribute
+   */
+  readonly portfolioId: string;
+}
+
+/**
+ * Properties for a Portfolio.
+ */
+export interface PortfolioProps {
+  /**
+   * Enforces a particular physical portfolio name.
+   */
+  readonly portfolioName: string;
+
+  /**
+   * The provider name.
+   */
+  readonly providerName: string;
```

Comment:
I took this name from the [original PR](https://github.com/aws/aws-cdk/pull/14991). However, I see that in the Console, this is called "Owner".

Is there any reason for this discrepancy? Maybe `owner` is a better name?

Reply:
`providerName` is better and is the prop also used by the [Cfn construct](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-servicecatalog-portfolio.html).  Since most of the usage is to facilitate portfolios to end users who actually use the portfolio using `owner` would convey meaning/responsibility that is not there.